### PR TITLE
patterns for continuous rates

### LINKED ISSUE #886: New patterns for rate of continuous process
Patterns for rate that is the 'speed' of a continuous process (e.g. reaction rate) and not a frequency/rate of occurrence .
Using http://purl.obolibrary.org/obo/PATO_0050001 and subclasses.

### PR DESCRIPTION
**Pull request by Clare72**
fixes #886

### CODE REVIEW

File: src/patterns/dosdp-dev/abnormalRateOfContinuousBiologicalProcess.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
+pattern_name: abnormalRateOfContinuousBiologicalProcess
+pattern_iri: http://purl.obolibrary.org/obo/upheno/patterns-dev/abnormalRateOfContinuousBiologicalProcess.yaml
+description: "Abnormal rate of a continuous biological process, e.g. rate of an enzyme reaction or growth rate. Does not include changes in rates of occurrence/frequency."
+
```
Comment (sbello) [inline src/patterns/dosdp-dev/abnormalRateOfContinuousBiologicalProcess.yaml:4]: Should add some examples from ontologies.
Would abnormal basal metabolism MP:0008961 (def. any anomaly in heat production of an organism at the lowest level of cell chemistry in an inactive, awake, and fasting state) fit this pattern?

Comment (sbello) [reply src/patterns/dosdp-dev/abnormalRateOfContinuousBiologicalProcess.yaml:4]: another example http://purl.obolibrary.org/obo/MP_0000233

File: src/patterns/dosdp-dev/abnormalRateOfContinuousBiologicalProcess.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  - biological_process
+- annotationProperty: exact_synonym
+  text: "abnormal speed of %s"
+  vars:
+  - biological_process
+- annotationProperty: exact_synonym
+  text: "changed continuous rate of %s"
+  vars:
+  - biological_process
+
```
Comment (sbello) [inline src/patterns/dosdp-dev/abnormalRateOfContinuousBiologicalProcess.yaml:48]: consider adding abnormal %s velocity as an exact synonym

File: src/patterns/dosdp-dev/abnormalRateOfContinuousBiologicalProcess.yaml (hunk 1 above)
Comment (ybradford) [reply src/patterns/dosdp-dev/abnormalRateOfContinuousBiologicalProcess.yaml:4]: example from ZP http://purl.obolibrary.org/obo/ZP_0000707 blood flow rate, abnormal

Comment (sbello) [reply src/patterns/dosdp-dev/abnormalRateOfContinuousBiologicalProcess.yaml:4]: HPO example http://purl.obolibrary.org/obo/HP_0012212

File: src/patterns/dosdp-dev/abnormalRateOfContinuousBiologicalProcess.yaml [hunk 3]
```diff
@@ -0,0 +1,52 @@
+pattern_name: abnormalRateOfContinuousBiologicalProcess
+pattern_iri: http://purl.obolibrary.org/obo/upheno/patterns-dev/abnormalRateOfContinuousBiologicalProcess.yaml
+description: "Abnormal rate of a continuous biological process, e.g. rate of an enzyme reaction or growth rate. Does not include changes in rates of occurrence/frequency."
```
Comment (rays22) [inline src/patterns/dosdp-dev/abnormalRateOfContinuousBiologicalProcess.yaml:3]: Contrast its usage to a PATO:0002242 velocity pattern.

File: src/patterns/dosdp-dev/abnormallyDecreasedRateOfContinuousBiologicalProcess.yaml [hunk 4]
```diff
@@ -0,0 +1,52 @@
+pattern_name: abnormallyDecreasedRateOfContinuousBiologicalProcess
+pattern_iri: http://purl.obolibrary.org/obo/upheno/patterns-dev/abnormallyDecreasedRateOfContinuousBiologicalProcess.yaml
+description: "Abnormally decreased rate of a continuous biological process, e.g. rate of an enzyme reaction or growth rate. Does not include decreases in rates of occurrence/frequency."
+
```
Comment (ybradford) [inline src/patterns/dosdp-dev/abnormallyDecreasedRateOfContinuousBiologicalProcess.yaml:4]: example from ZP http://purl.obolibrary.org/obo/ZP_0102933  digestion decreased rate, abnormal

**Review by ybradford — APPROVED** [review]
I've added some examples. 
ORCiD: 0000-0002-9900-7880

**Review by sbello — APPROVED** [review]
Looks good

**Pull request merged**